### PR TITLE
Stats page

### PR DESCRIPTION
**Pull request by HL3rd**
Completed stats page steps 5-7

### CODE REVIEW

File: src/main/webapp/js/stats-page-loader.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const urlParams = new URLSearchParams(window.location.search);
+const parameterUsername = urlParams.get('stats');
+
+/** Fetches stats and add them to the page. */
+function fetchStats(){
```
Comment (PandiyanA) [inline src/main/webapp/js/stats-page-loader.js]: Please add space before {.
e.g: 
function fetchStats() {

Please apply this comment to all the methods in this file and other files (if applicable).

**Review by PandiyanA — COMMENTED** [review]
Thanks for adding stats page. Please attach a screenshot of this page (to check out the UI).

**Comment by HL3rd** [comment]
image
Screenshot of Stats page

File: src/main/webapp/js/stats-page-loader.js (hunk 1 above)
Comment (PandiyanA) [reply src/main/webapp/js/stats-page-loader.js]: Addressed

**Review by PandiyanA — APPROVED** [review]
lgtm.

**Pull request merged**